pkg/apiserver/api_installer.go

```diff
@@ -49,16 +53,16 @@ func (a *APIInstaller) Install() (ws *restful.WebService, errors []error) {
 
 	// Initialize the custom handlers.
 	watchHandler := (&WatchHandler{
-		storage:                a.restHandler.storage,
-		codec:                  a.restHandler.codec,
-		canonicalPrefix:        a.restHandler.canonicalPrefix,
-		selfLinker:             a.restHandler.selfLinker,
-		apiRequestInfoResolver: a.restHandler.apiRequestInfoResolver,
+		storage: a.group.storage,
```

Comment:
Why doesn't it make sense to just pass a.group, as opposed to the individual fields? Because they're all private, currently?